common/app/views/fragments/inlineJSBlocking.scala.html

```diff
@@ -6,29 +6,32 @@
 
 @(page: model.Page)(implicit request: RequestHeader, context: model.ApplicationContext)
 
-<!--[if lt IE 9]>
-    <script src="@Static("javascripts/es5-html5.js")"></script>
-<![endif]-->
+@if(!mvt.PolyfillIO.isParticipating) {
+    <!--[if lt IE 9]>
+        <script src="@Static("javascripts/es5-html5.js")"></script>
+    <![endif]-->
+}
 
 @* NOTE the order of these includes is important  *@
 <script id="gu">
+    @if(!mvt.PolyfillIO.isParticipating) {
+        // ************* POLYFILLS *************
+        // Mostly handled by es5 shim above. Those which aren't are polyfilled ASAP.
 
-    // ************* POLYFILLS *************
-    // Mostly handled by es5 shim above. Those which aren't are polyfilled ASAP.
+        // It's faster to pass arguments in setTimeout than to use an anon function, but IE <10 can't do that.
+        // Used by RAF polyfill.
+        @Html(setTimeout().body)
 
-    // It's faster to pass arguments in setTimeout than to use an anon function, but IE <10 can't do that.
-    // Used by RAF polyfill.
-    @Html(setTimeout().body)
+        // Pollyfill requestAnimationFrame
+        @InlineJs(raf().body, "raf.js")
```

Comment:
Would we ever see JS using these polyfills in the body? Could we move them to above the things that use them in `inlineJSNonBlocking` instead of blocking?